frontend/lib/queries/dataset.ts

```
@@ -111,7 +108,12 @@ export async function searchDatasets(
     }>
   >(action, { ckanUrl: DMS });
 
-  return { ...res.result, datasets: res.result.results };
+  const datasets = res.result.results?.map((d) => ({
+    ...d,
+    dataset_type: !!d?.dashboard_url ? "dashboard" : "dataset",
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)